revaluate_client/src/components/Header/Header.css

```diff
@@ -0,0 +1,25 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap');
+.headerBar{
+    margin-left: 4.73%;
+    margin-right: 7.55%;
+    margin-top: 5.34%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
```

Comment:
I think can remove these too as flex will be dependent on the parent element size which is different across pages. Replace these with 'text-align: left'